libraries/fiff/fiff_coord_trans.cpp

```diff
@@ -162,6 +162,35 @@ bool FiffCoordTrans::read(QIODevice& p_IODevice, FiffCoordTrans& p_Trans)
 
 //=============================================================================================================
 
+void FiffCoordTrans::write(QIODevice &p_IODevice)
+{
+    //
+    //   Open the file, create directory
+    //
+
+    // Create the file and save the essentials
+    FiffStream::SPtr pStream = FiffStream::start_file(p_IODevice);
+    printf("Write coordinate transform in %s...\n", pStream->streamName().toUtf8().constData());
+    this->writeToStream(pStream.data());
+    pStream->close();
+}
+
+//=============================================================================================================
+
+void FiffCoordTrans::writeToStream(FiffStream* pStream)
+{
+    pStream->start_block(FIFFB_MEAS);
+    pStream->start_block(FIFFB_MEAS_INFO);
```

Comment:
Did you test what happens if you do not include FIFFB_MEAS and FIFFB_MEAS_INFO blocks? See [here](https://github.com/mne-tools/mne-cpp/blob/e8c2fc4206534447226f63040b985b5acb0a9c9d/libraries/mne/mne_bem.cpp#L361)

Reply:
No I did not. I'll write the test and will check then.